user_deprovision/README.md

```diff
@@ -10,11 +10,24 @@ This script is meant to be used as a command line tool with the following argume
 
 `./user_deprovision.py --access-token ENTER_PD_ACCESS_TOKEN --users-emails-from-csv ./emails-to-remove.csv --from-email user-requesting-deletion@example.com`
 
-**-a**, **--access-token**: A valid PagerDuty v2 REST API access token from your account
+**`-a`**, **`--access-token`**: A valid PagerDuty v2 REST API access token from your account
 
-**-u**, **--users-emails-from-csv**: A csv file of the PagerDuty email address(es) for the user(s) you want to delete from your account
+**`-u`**, **`--users-emails-from-csv`**: A csv file of the PagerDuty email address(es) for the user(s) you want to delete from your account
 
-**-f**, **--from-header**: The PagerDuty email address of the user that is requesting the deletion
+**`-f`**, **`--from-header`**: The PagerDuty email address of the user that is requesting the deletion
+
+### Optional arguments
+
+**`-v`**, **`--verbose`**: output the logs to the console while running
+
+**`-b`**., **`--backup`**: backup info about each user to a json file
+
+**`-y`**, **`--yes-to-all`**: by default the script will ask you before deleting _each user_, which can be tedious for 
```

Comment:
Flag name is actually defined as `--delete-yes-to-all`

Reply:
GOOD CATCH